File: src/main/java/com/commercehub/core/state/impl/xml/TokenTypes.java

package com.commercehub.core.state.impl.xml;
public interface TokenTypes {
    String ALL = "ALL";
    String SOME = "SOME";
    String NONE = "NONE";
    String ANY = "ANY";
    String AND = "AND";
    String OR = "OR";
    String NOT = "NOT";
    int TOK_ALL = 1;
    int TOK_SOME = 2;
    int TOK_NONE = 3;
    int TOK_ANY = 4;
    int TOK_AND = 5;
    int TOK_OR = 6;
    int TOK_NOT = 7;
    int TOK_LPAREN = 8;
    int TOK_RPAREN = 9;
    int TOK_COMMA = 10;
    int TOK_ID = 11;
    int TOK_FUNC = 12;
}

File: src/main/scala/pulleys/impl/xml/TokenTypes.scala
package pulleys.impl.xml

trait TokenTypes {
  val ALL: String = "ALL"
  val SOME: String = "SOME"
  val NONE: String = "NONE"
  val ANY: String = "ANY"
  val AND: String = "AND"
  val OR: String = "OR"
  val NOT: String = "NOT"
  val TOK_ALL: Int = 1
  val TOK_SOME: Int = 2
  val TOK_NONE: Int = 3
  val TOK_ANY: Int = 4
  val TOK_AND: Int = 5
  val TOK_OR: Int = 6
  val TOK_NOT: Int = 7
  val TOK_LPAREN: Int = 8
  val TOK_RPAREN: Int = 9
  val TOK_COMMA: Int = 10
  val TOK_ID: Int = 11
  val TOK_FUNC: Int = 12
}